reconcile/openshift_limitranges.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import semver
+import sys
+
+import utils.gql as gql
+import reconcile.openshift_base as ob
+
+from utils.openshift_resource import OpenshiftResource as OR
+from utils.defer import defer
+
+from reconcile.queries import NAMESPACES_QUERY
+
+QONTRACT_INTEGRATION = 'openshift-limitranges'
+QONTRACT_INTEGRATION_VERSION = semver.format_version(0, 1, 0)
+
+SUPPORTED_LIMITRANGE_TYPES = (
+    'default',
+    'defaultRequest',
+    'max',
+    'maxLimitRequestRatio',
+    'min'
+)
+
+
+def construct_resources(namespaces):
+    for namespace in namespaces:
+        # Get the linked limitRanges schema settings
+        limitranges = namespace.get("limitRanges", {})
+
+        body = {
+            'apiVersion': 'v1',
+            'kind': 'LimitRange',
+            'metadata': {
+                'name': limitranges['name'],
```

Comment:
I've added a check for the 'limitRanges' key and it will print a warning if it is missing.

It should not be missing though as I filter it when doing the graphql query. Check doesn't hurt though.